aws-emrserverless-application/aws-emrserverless-application.json

```diff
@@ -433,6 +437,11 @@
       "maxLength": 15,
       "pattern": "^[1-9][0-9]*(\\s)?(GB|gb|gB|Gb)$"
     },
+    "DiskType": {
+      "description": "Per worker DiskType resource. Shuffle optimized and Standard are only supported types and specifying diskType is optional",
```

Comment:
Do we want to say that it defaults to Standard when unspecified?

Reply:
These are generated by model.